index.bs

```diff
@@ -1309,15 +1309,66 @@ Verifiable Random Selection Procedure</h5>
 			The incomplete terms are assigned in result order.
 	</ol>
 
+<h5 id="AB-TAG-recall">
+Recalling AB or TAG members</h5>
+
+<h6 id="individual-removal">
+Individual Removal</h6>
+
+	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
+	can be <dfn export lt="remove|removal">removed</dfn> from those groups
+	if they are found by their peers
+	to be grossly neglecting their duties,
+	or to be acting in a way that seriously hampers the group's ability to function normally.
+
+	A chair of the [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold a hearing
+	on the potential [=removal=] of a participant
+	if requested by at least three of the participants in the group.
+	After giving the individual in question
+	an opportunity to defend themselves,
+	a vote on the proposed [=removal=] is held.
+	If at least three quarters of the participants in the group,
+	excluding the individual who is the subject of such vote,
+	then vote in favor of the proposal,
+	the individual's seat on [=AB=] or [=TAG=] is [=vacated=] immediately.
+
+<h6 id="collective-removal">
+Collective removal</h6>
+
+	The [=Advisory Committee=] <em class=rfc2119>may</em>
+	hold a <dfn export>vote of no confidence</dfn>
+	in the [=Advisory Board=] or the [=Technical Architecture Group=].
+
+	An [=Advisory Committee representative=] initiates a [=vote of no confidence=]
+	by sending a request to the Team, and <em class=rfc2119>should</em> also share this request with the Advisory Committee.
+	The request <em class=rfc2119>must</em> identify which of the [=AB=] or [=TAG=] is targeted,
+	and <em class=rfc2119>should</em> also include the rationale.
```

Comment:
I think that you want a three member threshold for this too.  Otherwise, this is open to trolling and DoS.

I'd be OK with a higher threshold than three, but not a lower one.

If that takes the form of one AC member initiating an override that has to be seconded by two other members in all cases, that would be ideal.  I know that this mechanism hasn't been activated, but it's an organizational vulnerability.

Reply:
It already is a 3 stage thing:
1. someone calls for a vote of no confidence
2. we check if >=5% of the membership agree that we should run a vote of no confidence (within a time limit of 1 week)
3. if so, we run the actual vote of no confidence

Adding an "at least two people need to agree" seems redundant with step two.

Reply:
I think that the second step is not feasible (you allow more time for the final vote, which only needs to clear the same threshold).  So I am suggesting a replacement for that stage.